lib/catcher/cache.rb

```diff
@@ -29,7 +29,20 @@ def keyed_response
     attr_reader :api
 
     def parsed_api_response
-      @parsed_api_response ||= Service.factory(api_resource, headers).parsed_api_response
+      @parsed_api_response ||= Service.factory(
```

Comment:
I'd be tempted to use just ```api``` as parameter (all you need is there) but you need to refactor the actual services:

```ruby
Service.factory(api).parse_api_response
```